### PR TITLE
add backup and restore node settings to configuration docs

### PR DESCRIPTION
**Pull request by mfelsche**

### CODE REVIEW

File: docs/configuration.txt [hunk 1]
```diff
@@ -534,6 +545,38 @@ Blobs
...
+  Only ``http``, ``https`` and ``ftp`` URLs need to be listed here for usage in
+  :ref:`ref-create-repository-types-url` repsoitories.
+  
+  ``file`` urls listed here must be prefixed with an entry configured in
+  :ref:`path.repo <conf-path-repo>`.
+
+.. _conf-repositories-url-supported-protocols:
+
+**repositories.url.supported_protocols**
+  | *Default:* ``http``, ``https``, ``ftp``, ``file`` and ``jar``
```
Comment (seut) [inline docs/configuration.txt]: either be sure what 'jar' means and that it works or remove it here and change the defaults

File: docs/configuration.txt [hunk 2]
```diff
@@ -438,6 +438,17 @@ Paths
        file:
          file: ${path.logs}/${cluster.name}.log
 
+.. _conf-path-repo:
+
+**path.repo**
+  | *Runtime:* ``no``
+
+  A list of filesystem or UNC paths where repositories of type
+  :ref:`ref-create-repository-types-fs` may be stored.
```
Comment (seut) [inline docs/configuration.txt]: as said, a note why (security concerns) this is necessary would be nice

**Comment by seut** [comment]
otherwise ack

File: docs/configuration.txt (hunk 2 above)
Comment (mfelsche) [inline docs/configuration.txt]: done

File: docs/configuration.txt (hunk 1 above)
Comment (mfelsche) [inline docs/configuration.txt]: jar urls are there for read-only access to jar files.
format is ``jar:<url-to-jar-file>!/<path-in-jar>``
see for example here.

**Pull request closed without merging**